src/Symfony/Component/DependencyInjection/Alias.php

```diff
@@ -15,11 +15,11 @@
 
 class Alias
 {
-    private $id;
-    private $public;
-    private $deprecation = [];
+    private string $id;
+    private bool $public;
+    private array $deprecation = [];
 
-    private static $defaultDeprecationTemplate = 'The "%alias_id%" service alias is deprecated. You should stop using it, as it will be removed in the future.';
+    private static string $defaultDeprecationTemplate = 'The "%alias_id%" service alias is deprecated. You should stop using it, as it will be removed in the future.';
```

Comment:
I'd rather turn this into a `private const` (which was probably not possible at the time it was added)

Reply:
We have the similar one in Definition

Reply:
#41933